backend/src/App.js

```diff
@@ -20,14 +22,18 @@ export const app = async () => {
   const userRegistrationDetailsController = new UsersRegistrationController(
     userRegistrationDetailsService,
   );
+  const [boardsController, boardsService] = boardsModule(repositoryType);
+  const [announcementController] = announcementsModule(repositoryType, boardsService);
 
   const restApiServer = express();
   restApiServer.use(cors());
   restApiServer.use(express.json());
   restApiServer.use(express.urlencoded({ extended: true }));
   restApiServer.use(morgan('combined'));
   restApiServer.use('/rest-api', userRegistrationDetailsController.router);
-  restApiServer.use('/rest-api', commentModule(repositoryType));
+  restApiServer.use('/rest-api', CommentModule(repositoryType));
```

Comment:
Widzę, że comment jest funkcją, to można od razu zmienić na małą literę :)
Ooo, to wiąże się z tym, o czym rozmawialiśmy. Czyli, że nie twój moduł, ale to można zmienić, bo korzystasz z tego :))))

```suggestion
  restApiServer.use('/rest-api', commentModule(repositoryType));
```

Reply:
done